test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -2446,6 +2459,34 @@ static void crypto_test_check_alg_sha512(void)
 		  false);
 }
 
+static odp_queue_t sched_compl_queue_create(void)
+{
+	odp_queue_param_t qparam;
+
+	odp_queue_param_init(&qparam);
+	qparam.type = ODP_QUEUE_TYPE_SCHED;
+	qparam.sched.prio  = ODP_SCHED_PRIO_DEFAULT;
+	qparam.sched.sync  = ODP_SCHED_SYNC_PARALLEL;
+	qparam.sched.group = ODP_SCHED_GROUP_ALL;
+
+	return odp_queue_create("crypto-out", &qparam);
+}
+
+static odp_queue_t plain_compl_queue_create(void)
+{
+	return odp_queue_create("crypto-out", NULL);
+}
+
+static odp_event_t sched_compl_queue_deq(void)
+{
+	return odp_schedule(&suite_context.queue, ODP_SCHED_NO_WAIT);
```

Comment:
schedule() call does not need queue parameter (should use NULL instead). Passing the suite queue handle there is risky, as on an error/bug the call could overwrite some other queue handle (or invalid) there.

Reply:
Ack. Changed the queue parameter passed to NULL in v5.